docs/languages/julia.md

```diff
@@ -0,0 +1,71 @@
+    1. Start VS Code.
+    2. Inside VS Code, go to the extensions view by
+        clicking `View` on the top tool bar and then selecting `Extensions`.
+    3. In the extensions view, simply search for the term ``julia`` in the marketplace
+        search box, then select the extension named ``Julia`` (as shown below) and click the install button.
+        You should restart VS Code after this step. Note that if you run into any issues installing the Julia VS Code Extension, check out the [docs on installing an extension](https://code.visualstudio.com/docs/editor/extension-marketplace) which should help clarify any issues.
+        
+    ![Julia in the VS Code Marketplace](images/julia/julia-extension-marketplace.png)
+
+## Running Code
+
+There are a variety of different ways to run Julia code through the VS Code IDE. You have the option to run a Julia file (via `(Ctrl+F5)`, which will run whatever Julia file you have open and active), execute Julia commands via the REPL, or even just execute a specific block of code from a file you have open. To learn more about these options, head to the [The Julia VS Code Docs section on running code](https://www.julia-vscode.org/docs/stable/userguide/runningcode/).
+
+## Debugging 
+
+You can start debugging by opening the Julia file which you would like to debug. Then, click the debug icon on the left (as shown below):
+
+![Getting started debugging Julia code](images/julia/debug1.png)  
+
+Next, you can add a breakpoint by clicking to the left of the line number:
+
+![Adding a breakpoint](images/julia/debug2.png)  
+
+Note that the red dot will not show up until after you have clicked next to the line number. 
+
+After you have a breakpoint added (or any other kind of debug configuration), click the "Run and Debug" button on the left. It may take a few seconds for the initial run to begin. You should then see the output of running the code with the debug configuration. In this example, since we added a breakpoint, you will see the following: 
+
+![Run and Debug your Julia file](images/julia/debug3.png)  
+
+Notice that the second print command has yet to execute given we cannot see the text from that proint command in the terminal yet. You can finish the execution of the program by clicking the "Countinue" button:
+
+![Finish the code execution in the debugger](images/julia/debug4.png)  
+
+To find out more about debugging Julia code with VS Code, head to [The Julia VS Code Docs section on Debugging](https://www.julia-vscode.org/docs/stable/userguide/debugging/). 
+
+## Code completion (IntelliSense)
+
+The Julia VS Code extension comes with Code completion thanks to IntelliSense. This feature works out of the box and is extremely useful for experienced and beginner Julia developers alike.
+
+![Code completion with IntelliSense](images/julia/code-completion.gif)
```

Comment:
This is a older .gif which has the old Activity bar icons. Could you redo this .gif? 
It would also be good to use Dark theme to match the other images.
This image also has "Spell Enabled" in the Status bar. Better to have a very clean installation for screenshots and .gifs, and only include the Julia extension and any dependencies.